dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,761 @@
+=== Allowing Pods to Reference Images Across Projects
+
+When using the internal registry, to allow pods in *project-a* to reference
+images in *project-b*, a service account in *project-a* must be bound to the
+`system:image-puller` role in *project-b*:
+
+----
+$ oc policy add-role-to-user \
+    system:image-puller system:serviceaccount:project-a:default \
+    --namespace=project-b
+----
+
+After adding that role, the pods in *project-a* that reference the default
+service account will be able to pull images from *project-b*.
+
+To allow access for any service account in *project-a*, use the group:
+
+----
+$ oc policy add-role-to-group \
+    system:image-puller system:serviceaccounts:project-a \
+    --namespace=project-b
+----
+
+[[allowing-pods-to-reference-images-from-other-secured-registries]]
+=== Allowing Pods to Reference Images from Other Secured Registries
+
+The *_.dockercfg_* file (or *_$HOME/.docker/config.json_* for newer Docker
+clients) is a Docker credentials file that stores your information if you have
+previously logged into a secured or insecure registry.
+
+To pull a secured Docker image that is not from {product-title}'s internal
+registry, you must create a _pull secret_ from your Docker credentials and add
+it to your service account.
+
+If you already have a *_.dockercfg_* file for
+the secured registry, you can create a secret from that file by running:
+
+----
+$ oc secrets new <pull_secret_name> .dockercfg=<path/to/.dockercfg>
+----
+
+Or if you have a *_$HOME/.docker/config.json_* file:
+
+----
+$ oc secrets new <pull_secret_name> .dockerconfigjson=<path/to/.docker/config.json>
+----
+
+If you do not already have a Docker credentials file for the secured registry,
+you can create a secret by running:
+
+----
+$ oc secrets new-dockercfg <pull_secret_name> \
+    --docker-server=<registry_server> --docker-username=<user_name> \
+    --docker-password=<password> --docker-email=<email>
+----
+
+To use a secret for pulling images for pods, you must add the secret to your
+service account. The name of the service account in this example should match
+the name of the service account the pod will use; *default* is the default
+service account:
+
+----
+$ oc secrets add serviceaccount/default secrets/<pull_secret_name> --for=pull
+----
+
+To use a secret for pushing and pulling build images, the secret must be
+mountable inside of a pod. You can do this by running:
+
+----
+$ oc secrets add serviceaccount/builder secrets/<pull_secret_name>
+----
+
+[[importing-tag-and-image-metadata]]
+== Importing Tag and Image Metadata
+
+An image stream can be configured to import tag and image metadata from an image
+repository in an external Docker image registry. You can do this using a few
+different methods.
+
+- You can manually import tag and image information with the `oc import-image`
+command using the `--from` option:
++
+----
+$ oc import-image <image_stream_name> --from=<docker_image_repo> --confirm
+----
```

Comment:
The `spec.dockerImageRepository` is the "old" way of importing images. What `oc import-image` does it  the new recommended way of importing images, because you have a more granular control over the tags. Having said that, both are valid, but per-tag is the preferred one. I was thinking about making a note here about that while reading it just now.